src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -245,6 +245,21 @@ def set_reboot_pending(self, is_reboot_pending):
         self.is_reboot_pending = is_reboot_pending
     # endregion
 
+    # region - Terminal state management
+    def report_sequence_number_changed_termination(self):
+        """ Based on the current operation (Assessment or Installation), adds an error status and sets the substatus to error """
+        current_operation = self.execution_config.operation.lower()
+        error_code = Constants.PatchOperationErrorCodes.NEWER_OPERATION_SUPERSEDED
+        message = "Execution was stopped due to a newer operation taking precedence."
+
+        if current_operation == Constants.ASSESSMENT.lower() or self.execution_config.exec_auto_assess_only:
+            self.add_error_to_status(message, error_code, current_operation_override_for_error=Constants.ASSESSMENT)
+            self.set_assessment_substatus_json(status=Constants.STATUS_ERROR)
```

Comment:
Why remove CP from here?

Reply:
NOTE: There is some ambiguity on pre-emption between different operations, which needs more investigation that will happen outside of this PR. If any changes are discovered during those, they will be in separate PR. The code coverage check for ConfigurePatching in lines #259-261 is noted and will be looked into during this investigation